docs/.link-check-errors.txt

```diff
@@ -1,2 +1,3 @@
 Broken links found by `docs:link-check` that exist:
 
+- https://mui.com/x/react-data-grid/row-grouping/#disable-the-row-grouping
```

Comment:
this is in a blog post from the monorepo, so I'll have to open a separate PR over there to fix it